### ISSUE TITLE
fetch_posts breaks when fetching suspended accounts

### ISSUE THREAD

**Issue by plygrnd** [plygrnd/tinkerbell#1]
Suspended accounts don't have the created_utc property, so fetch_posts will throw an AttributeError when fetching them:

```
In [3]: s.redditor('kikeslasher').created_utc
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-3-1eafab403aae> in <module>()
----> 1 s.redditor('kikeslasher').created_utc

~/projects/tinkerbell/lib/python3.6/site-packages/praw/models/reddit/base.py in __getattr__(self, attribute)
     30         if not attribute.startswith('_') and not self._fetched:
     31             self._fetch()
---> 32             return getattr(self, attribute)
     33         raise AttributeError('{!r} object has no attribute {!r}'
     34                              .format(self.__class__.__name__, attribute))

~/projects/tinkerbell/lib/python3.6/site-packages/praw/models/reddit/base.py in __getattr__(self, attribute)
     32             return getattr(self, attribute)
     33         raise AttributeError('{!r} object has no attribute {!r}'
---> 34                              .format(self.__class__.__name__, attribute))
     35 
     36     def __hash__(self):

AttributeError: 'Redditor' object has no attribute 'created_utc'
```

We can fix this by adding a check for is_suspended thus:

```
In [4]: s.redditor('kikeslasher').is_suspended
Out[4]: True
```

**Comment by plygrnd** [comment]
This might be an opportunity to admit defeat and actually write some integration tests.

**Comment by plygrnd** [comment]
(╯°□°）╯︵ ┻━┻
https://www.reddit.com/user/beefythecat/about.json
https://www.reddit.com/user/sad__sack/about.json
Notice the difference!?
Both are valid, yet I have 300x more data on my profile than them. Bloody inconsistent API/database/whatever.

**Comment by plygrnd** [comment]
That was a pain in the arse to figure out. Fixed in 8b3bf29.
